opencga-catalog/src/main/java/org/opencb/opencga/catalog/exceptions/CatalogDBException.java

```diff
@@ -133,4 +133,9 @@ public static CatalogDBException cannotUpdateMultipleEntries(String field, Strin
         return new CatalogDBException("Update " + entry + ": Cannot update '" + field + "' parameter for multiple entries");
     }
 
+    public static CatalogDBException jwtSecretKeyException() {
+        return new CatalogDBException("JWT secret key should be at least 30 characters long and contain at least 1 upper case, 1 lower "
```

Comment:
you should use here the SECRET_KEY_MIN_LENGTH

Reply:
fixed